tests/integration/Entries/ImportEntryTest.cs

```diff
@@ -38,29 +38,6 @@ private FileParameter GetFileParameter()
             return new FileParameter(fileStream, "test.pdf", "application/pdf");
         }
 
-        [TestMethod]
-        public async Task DocumentCreated()
```

Comment:
Example: the logic of this test is already included in the other test: `DocumentCreatedWithTemplate()`